frontend/src/v5/store/auth/auth.sagas.ts

```diff
@@ -98,8 +98,19 @@ function* logout() {
 	yield put(AuthActions.setPendingStatus(false));
 }
 
+function* kickedOut() {
+	yield put({ type: 'RESET_APP' });
+	yield put(DialogsActions.open('warning', {
+		message: formatMessage({
+			id: 'auth.logout.kickedOut',
+			defaultMessage: 'You have been logged out because your account has signed in elsewhere',
```

Comment:
Looking at the README in `src\v5\services\chat\README.md` it seems to suggest that the logged out event returns an object containing a _reason_. If so, would it be useful to pass the _reason_ into this message in the case that there are multiple reasons a user is kicked out?